backend/services/connection_service.go

```diff
@@ -218,8 +228,10 @@ func (c *connectionService) buildOption(config types.ConnectionConfig) (*redis.O
 		} else {
 			option.Dialer = dial
 		}
-		option.ReadTimeout = -2
-		option.WriteTimeout = -2
+		if config.SSH.Enable {
```

Comment:
prevent the socks5 tunnel timeout with the new go-redis sdk (v1.14.0 -> v1.17.0)